workspace/TS100/Core/BSP/Pine64/Vendor/SoC/gd32vf103/Common/Source/Drivers/gd32vf103_timer.c

```diff
@@ -980,7 +979,7 @@ void timer_channel_output_fast_config(uint32_t timer_periph, uint16_t channel, u
     \param[in]  channel:
                 only one parameter can be selected which is shown as below:
       \arg        TIMER_CH_0: TIMER channel 0(TIMERx(x=0..4))
-      \arg        TIMER_CH_1: TIMER channel 1(TIMERx(x=0..4))
+      \arg        TIMER_CH_1: TIMER channel 1(TIMERx(x=0..41))

```

Comment:
I do not really understand those changes to the comments. I mean in general in this file.

I was trying to check those things with the https://files.pine64.org/doc/datasheet/pinecil/GD32VF103_Datasheet_Rev%201.1.pdf and it looks like that original comments were correct everywhere in the file (GD32VF103 have 4 generic timers and not 41).

Reply:
I have changed from using the drivers published on the gigadevice website to the ones from NMSIS

Moved to this one : https://github.com/Nuclei-Software/nuclei-sdk

So I didnt do any of these changes, but rather just moved to their version of the files. (all the gd32vf103_* files are direct from that repo)